packages/test/local-server-tests/src/test/opsOnReconnect.spec.ts

```diff
@@ -654,7 +655,10 @@ describe("Ops on Reconnect", () => {
 			// At this point, the delta manager should have the messages
 			// in its buffer but not in its outbound queue,
 			// as ops have not been flushed yet
-			assert.strictEqual(container1.deltaManager.outbound.length, 0);
+			assert.strictEqual(
+				(container1.deltaManager as IDeltaManagerInternal).outbound.length,
```

Comment:
Please avoid use of `as` cast. It is very unsafe in many situations. Failing other ways to know this is internal version, I recommend having a helper that takes care of the cast. That helper can then ensure at least going from `IDeltaManager<...>`.

Reply:
```suggestion
			assert(isDeltaManagerInternal(container1.deltaManager));
			assert.strictEqual(
				container1.deltaManager.outbound.length,
```

Reply:
That should also work, but it will be a lot of extra lines compared to cast helper.

Reply:
Fixed it.